guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -71,7 +71,7 @@ When your application is not in the production mode, you can set Magento to comp
 
 To set the compilation mode, do the following:
 <ol>
-<li>In the Magento Admin, navigate to <b>Stores</b> > <b>Configuration</b> > ADVANCED > <b>Developer</b>.</li>
+<li>In the Magento Admin, navigate to <b>Stores</b> > <b>Configuration</b> > <b>ADVANCED</b> > <b>Developer</b>.</li>
```

Comment:
In this page there are multiple HTML-formatted lists. Please convert them to markdown lists: https://daringfireball.net/projects/markdown/syntax#list